netty/build.gradle

```diff
@@ -5,7 +5,18 @@ dependencies {
             libraries.netty
 
     // Tests depend on base class defined by core module.
-    testCompile project(':grpc-core').sourceSets.test.output
+    testCompile project(':grpc-core').sourceSets.test.output,
+                project(':grpc-testing')
+}
+
+test {
+    jvmArgs "-Xbootclasspath/p:" + configurations.alpnboot.asPath
+    testLogging {
+        exceptionFormat "full"
```

Comment:
I know that exceptionFormat is already being set to "full" in the root build.gradle. It seems like if these other options are useful, we would want them at the root as well.